doc/faq.rst

```diff
@@ -14,15 +14,24 @@ for threaded applications.
 Is PyMongo fork-safe?
 ---------------------
 
-PyMongo is not fork-safe. Care must be taken when using instances of
-:class:`~pymongo.mongo_client.MongoClient` with ``fork()``. Specifically,
-instances of MongoClient must not be copied from a parent process to
-a child process. Instead, the parent process and each child process must
-create their own instances of MongoClient. Instances of MongoClient copied from
-the parent process have a high probability of deadlock in the child process due
-to the inherent incompatibilities between ``fork()``, threads, and locks
-described :ref:`below <pymongo-fork-safe-details>`. PyMongo will attempt to
-issue a warning if there is a chance of this deadlock occurring.
+Starting in PyMongo 4.3, forking on a compatible Python interpreter while
+using a client will result in all locks held by :class:`~bson.objectid
+.ObjectId` and :class:`~pymongo.mongo_client.MongoClient` being released.
+In addition, instances of :class:`~pymongo.topology.Topology` will be
+replaced with identical settings. This feature is only available on
+interpreters where :meth:`os.register_at_fork` is available, such as CPython
+3.7+ on Unix.
+
+Earlier versions of PyMongo are not fork-safe. Care must be taken when using
+instances of :class:`~pymongo.mongo_client.MongoClient` with ``fork()``.
+Specifically, instances of MongoClient must not be copied from a parent
+process to a child process. Instead, the parent process and each child
+process must create their own instances of MongoClient. Instances of
+MongoClient copied from the parent process have a high probability of
+deadlock in the child process due to the inherent incompatibilities between
+``fork()``, threads, and locks described :ref:`below
+<pymongo-fork-safe-details>`. PyMongo will attempt to issue a warning if
+there is a chance of this deadlock occurring.
```

Comment:
Should we just remove all the outdated sections now that pymongo is fork safe? The old non-fork safe behavior will still be readable on the docs for old versions on pymongo.